### ISSUE TITLE
com.swmansion.reanimated.AndroidErrorHandler.raise  (AndroidErrorHandler.java) v2.3.0-beta.2

### ISSUE THREAD

**Issue by pronevich** [software-mansion/react-native-reanimated#2509]
## Description

Stack trace from Crashlytics, Android 11, Redmi Note 9 Pro:

```
Fatal Exception: java.lang.RuntimeException: java.lang.AssertionError: Illegal type provided
       at com.swmansion.reanimated.AndroidErrorHandler.raise(AndroidErrorHandler.java:6)
       at com.swmansion.reanimated.Scheduler.triggerUI(Scheduler.java)
       at com.swmansion.reanimated.Scheduler$1.run(Scheduler.java:23)
       at com.swmansion.reanimated.Scheduler$2.runGuarded(Scheduler.java:41)
       at com.facebook.react.bridge.GuardedRunnable.run(GuardedRunnable.java:31)
       at android.os.Handler.handleCallback(Handler.java:938)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:236)
       at android.app.ActivityThread.main(ActivityThread.java:7861)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:600)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:967)
```

## Package versions

- React Native: 0.65.1
- React Native Reanimated: 2.3.0-beta.2

## Affected platforms

- [x] Android

**Comment by BubbleTrouble14** [comment]
@pronevich Did you get it any Reanimated 2 version working for the Redmi Note 9 Pro ? As I created an app and uploaded it to the playstore and somebody with the same device gets this crash.

**Comment by pronevich** [comment]
@BubbleTrouble14 stable version works fine, this issue happened also in other devices for beta version

**Comment by BubbleTrouble14** [comment]
@pronevich So version 2.2.4 should work ?

**Comment by pronevich** [comment]
@BubbleTrouble14 yeah 2.2.4 works fine for us on Android

**Comment by BubbleTrouble14** [comment]
@pronevich Hmm weird. Could you maybe test and see if my app runs on your Redmi Note 9 Pro ?

**Comment by pronevich** [comment]
@BubbleTrouble14 I don't have this device, it's stack trace for this device from Crachytics for some user. In stable 2.2.4 no issues were reported per hundreds of different devices.

**Comment by BubbleTrouble14** [comment]
Ah okay. Thx for you help !

**Comment by tomekzaw** [comment]
Looks similar to #2568

**Comment by BubbleTrouble14** [comment]
It actually wasnt due to reanimated 2. Was due to my translations which would try get the device language but was null on some devices causing the inconsistend crashes on different devices.

**Comment by tomekzaw** [comment]
Closed due to existing issue: #2568
